contracts/ZkSync_SpokePool.sol

```diff
@@ -118,14 +119,5 @@ contract ZkSync_SpokePool is SpokePool {
         emit SetZkBridge(address(_zkErc20Bridge), oldErc20Bridge);
     }
 
-    // L1 addresses are transformed during l1->l2 calls.
-    // See https://github.com/matter-labs/era-contracts/blob/main/docs/Overview.md#mailboxfacet for more information.
-    // Another source: https://github.com/matter-labs/era-contracts/blob/41c25aa16d182f757c3fed1463c78a81896f65e6/ethereum/contracts/vendor/AddressAliasHelper.sol#L28
-    function _applyL1ToL2Alias(address l1Address) internal pure returns (address l2Address) {
```

Comment:
Just to triple check, arbitrum and zksync do this the exact same way, right? Code looks the same, but just want to sanity check that I read this right.

Reply:
Yeah. (Just to enumerate) Arbitrum and ZkSync both have the same aliasing method, and all other chains query the messenger contract.

Reply:
Perfect.